src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/magiclink/MagicLinkEmailSender.kt

```diff
@@ -10,12 +10,61 @@ class MagicLinkEmailSender(
   private val javaMailSender: JavaMailSender,
 ) {
 
+  private companion object {
+    val TEXT_BODY_TEMPLATE: String = """
```

Comment:
Whilst the plain text and html versions aren't radically different, this does pave the way for being able to add basic formatting to the html version of the message.

Having the 2 templates here as strings and using `String.format` is OK for now, but if they start to get any more complex perhaps we could consider a proper templating library. I don't know what we generally use for this type of thing in MoJ projects, but given this is a spring boot project, thymeleaf might be a reasonable option to consider?